Source/UI/Test Results/IPFTestResult.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+@objcMembers
+final class IPFTestResult: NSObject {
```

Comment:
Can't we just reuse the existing Objective-C definition here?

Reply:
Actually this is a new object to save the test result (date, mode, speed etc) of each test. There is no existing objective-c definition for the same. Also I just switched this locally to a swift struct and made it codable so we can serialize to property list and save to disk